crates/bevy_ecs/src/schedule/schedule.rs

```diff
@@ -18,7 +18,7 @@ use fixedbitset::FixedBitSet;
 use crate::{
```

Comment:
Note that you will hit a roadblock when porting `bevy_render`: in #7046 (which pipelined rendering depends on) we added `remove_stage`, whose analog would be `remove_set` in stageless. We have explicitly decided not to support removal ops in stageless as they can invalidate upstream invariants, so we will need to discuss alternative "extract stage" approaches here. @hymm @james7132.

Reply:
It is still possible to remove a `Schedule` from the `Schedules` resource. Can extraction just be another schedule?

Reply:
Hmm yeah I think that would probably work in this case.

Reply:
Yeah. I was just expecting to use a full schedule for this. Does apply_buffers exist on a schedule?

Reply:
IIRC, it automatically applies all unapplied systems at the end of execution. Not sure if we can persist the unapplied system state and defer it until later/transfer it to another thread in the current construction.

Reply:
So that's what all those stage methods were for. We can reintroduce them on schedule.

Reply:
The initial work has been done now, and `Schedule::apply_buffers` has been added.